lib/services/response.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:wildlife_api_connection/api_client.dart';
+import 'package:wildlife_api_connection/response_api.dart';
+
+class ResponseService {
+  final _responseApi = ResponseApi(
+    ApiClient(
+        "https://wildlifenl-uu-michi011.apps.cl01.cp.its.uu.nl/interaction"),
+  );
```

Comment:
Make use of apiconfig instead of hardcoded baseurl. Developers expect to change baseurl by updating .env properties.
This may cause a debugging session where the developer ruled out the probability that the baseurl is not updated.

Suggested fix:
  final _responseApi = ResponseApi(
    AppConfig.shared.apiClient, (maybe add +/interaction)
  );
  
  Don't know why the '/interaction' is added but you might need to work around that since the baseurl should only be the base and no deeper routes.

Reply:
I checked in the package, everywhere it adds the needed path automatically. For some reason it was not update to all the services yet. I have updated it now.